static/app/components/actions/archive.tsx

```diff
@@ -115,10 +115,8 @@ function ArchiveActions({
     <ButtonBar className={className} merged>
       <ArchiveButton
         size={size}
-        tooltipProps={{delay: 300, disabled: disabled || disableTooltip}}
-        title={t(
-          'Silences alerts for this issue and removes it from the issue stream by default.'
-        )}
+        tooltipProps={{delay: 1000, disabled: disabled || disableTooltip}}
+        title={t('Hides the issue until sh*t hits the fan and events escalate.')}
```

Comment:
```suggestion
        title={t('Hides the issue until the sh*t hits the fan and events escalate.')}
```

How it is isn't wrong, but my brain stumbled over it the first time I read it because I'm so used to the "the" being there. An argument could be made either way - just my $.02.

Reply:
yah that's fine with me